1/index.html

```diff
@@ -9,10 +9,11 @@
   </head>
```

Comment:
Could you change from <html lang="en"> to <html lang="ja"> ?

Reply:
Does this mean that <head> is not necessary?

Reply:
Sorry, I accidentally sent an incomplete message;<

Could you change from <html lang="en"> to <html lang="ja"> ?

Reply:
understood.
The change from html lang="en" to html lang="ja" has been completed.
a167462b116e70121b2299c0c27fab3e0594359b